streams/src/main/java/org/apache/kafka/streams/kstream/internals/KStreamImpl.java

```diff
@@ -685,13 +685,13 @@ public boolean test(final K1 key, final V1 value) {
                                                                                     windows.maintainMs());
 
             KStreamKStreamJoin<K1, R, V1, V2> joinThis = new KStreamKStreamJoin<>(otherWindow.name(),
-                                                                                  windows.before,
                                                                                   windows.after,
+                                                                                  windows.before,
```

Comment:
Just to confirm, you are asking me to revert to the previous semantics and to fix the JoinWindows documentation to match those, yes?